.github/workflows/ci.yml

```diff
@@ -1,30 +1,127 @@
----
-
 name: CI
 
 on:
   pull_request:
-    branches: [devel]
-
   push:
-    branches: [devel]
+
```

Comment:
Did you intentionally add this new line?

Reply:
Yes, I have two blank lines between sections in the document as I added another here to match.